core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java

```diff
@@ -103,6 +108,11 @@ public LocalCacheFileInStream(URIStatus status, FileInStreamOpener fileOpener,
       mCacheContext = CacheContext.defaults();
     }
     Metrics.registerGauges();
+
+    mBufferSize = (int) conf.getBytes(PropertyKey.USER_CLIENT_CACHE_IN_STREAM_BUFFER_SIZE);
+    if (mBufferSize > 0) {
```

Comment:
What would happen if `mBufferSize` is <= 0? Will the code throw NPE at some point in that case? If that's the case, instead of `if`, use `checkArgument` (better to throw here)

Reply:
add a checkArgument as you said, but I still keep the 'if' because it should handle the case when the buffer size equals to 0.